linux_install.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+mkdir /home/.sms
```

Comment:
Shouldn't this be `/home/$USER/.sms` here and in other places?

Reply:
Fixed it.